e3db/src/main/java/com/tozny/e3db/LocalEAKInfo.java

```diff
@@ -87,11 +87,8 @@ public String encode() {
     doc.put("signer_signing_key", getSignerSigningKey());
     doc.put("eak", getKey());
 
-    try {
-      return mapper.writeValueAsString(doc);
-    } catch (JsonProcessingException e) {
-      throw new RuntimeException(e);
-    }
+    return mapper.writeValueAsString(doc);
```

Comment:
I find it curious how much of this is actually removing some try/catch blocks so the error doesn't get turned into a RuntimeException.